examples/produce_part_brep/README.md

```
@@ -0,0 +1,22 @@
+# /pub/examples/partcad/produce_part_brep
+
+PartCAD example project to demonstrate parts defined using BREP files.
+
+## Usage
+```shell
+pc inspect box
+```
+
+
```

Markdownlint (0.37.0)

Expected: 1; Actual: 2
Multiple consecutive blank lines

(MD012, no-multiple-blanks)